x/shared/session.go

```diff
@@ -68,3 +68,19 @@ func GetSessionGracePeriodEndHeight(sessionEndHeight int64) int64 {
 func IsGracePeriodElapsed(sessionEndHeight, currentHeight int64) bool {
 	return currentHeight > GetSessionGracePeriodEndHeight(sessionEndHeight)
 }
+
+// GetClaimWindowOpenHeight returns the block height at which the claim window opens.
+// This height is *inclusive*, meaning that if all other offsets were 0, this is the
+// earliest height at which a claim could be committed.
+func GetClaimWindowOpenHeight(sharedParams *sharedtypes.Params, sessionEndHeight int64) int64 {
+	return GetSessionGracePeriodEndHeight(sessionEndHeight) +
+		int64(sharedParams.GetClaimWindowOpenOffsetBlocks()) + 1
+}
+
+// GetClaimWindowCloseHeight returns the block height at which the claim window closes.
+// This height is *inclusive*, meaning that if all other offsets were 0, this is the
+// latest height at which a claim could be committed.
+func GetClaimWindowCloseHeight(sharedParams *sharedtypes.Params, sessionEndHeight int64) int64 {
+	return GetSessionGracePeriodEndHeight(sessionEndHeight) +
+		int64(sharedParams.GetClaimWindowOpenOffsetBlocks()) + 1
```

Comment:
Double check the `+ 1` (will have an effect on other potential off-by-one locations).